WalletWasabi/Blockchain/Analysis/FeesEstimation/HybridFeeProvider.cs

```diff
@@ -141,7 +115,7 @@ private bool SetAllFeeEstimateIfLooksBetter(AllFeeEstimate? fees)
 		var current = AllFeeEstimate;
 		if (fees is null
 			|| fees == current
-			|| (current is not null && ((!fees.IsAccurate && current.IsAccurate) || fees.Estimations.Count <= current.Estimations.Count)))
+			|| (current is not null && fees.Estimations.Count <= current.Estimations.Count))
```

Comment:
All this code smells so bad that it is a miracle that it works but I don't want to change it because it is out of scope. I think I will be forced to remove things in https://github.com/zkSNACKs/WalletWasabi/pull/12400 anyway.